README.md

```diff
@@ -30,7 +30,9 @@
 
 # Fingerprint Pro Server API Java SDK
 
-[Fingerprint](https://fingerprint.com/) is a device intelligence platform offering 99.5% accurate visitor identification. The Fingerprint Server Java SDK is an easy way to interact with the Fingerprint [Server API](https://dev.fingerprint.com/reference/pro-server-api) from your Java application. You can retrieve visitor history or individual identification events.
+**Fingerprint Pro Server API allows you to get information about visitors and about individual events in a server environment.
```

Comment:
Why this change? The old description is more consistent with the Integrations page, [documentation](https://dev.fingerprint.com/docs/java-server-api-sdk) and our guidelines.